_posts/2019-04-08-lecture-22.md

```diff
@@ -0,0 +1,339 @@
+---
+layout: distill
+title: "Lecture 22: Bayesian non-parametrics"
+description: An introduction to Bayesian non-parametrics and the Dirichlet process.
+date: 2019-04-08
+
+lecturers:
+  - name: Eric Xing
+    url: "https://www.cs.cmu.edu/~epxing/"
+
+authors:
+  - name: Yao Chong Lim  # author's full name
+    url: "#"  # optional URL to the author's homepage
+  - name: Yue Wu
+    url: "#"
+  - name: Dongshunyi Li
+    url: "#"
+  - name: Steve Landers
+    url: "#"
+
+editors:
+  - name: Maruan Al-Shedivat  # editor's full name
+    url: "#"  # optional URL to the editor's homepage
+
+---
+
+## Motivation
+
+### Clustering
+Bayesian non-parametrics is one a very popular topic in machine learning. The topic is not as popular right now but it is still useful since it's connected to a wide range of techniques we use today; for example, Gaussian Process can be applied to DNNs to make the layers infinitely wide.
```

Comment:
```suggestion
Bayesian non-parametrics used to be a popular topic in machine learning. The topic is not as popular right now but it is still useful since it's connected to a wide range of techniques we use today; for example, Gaussian Process can be applied to DNNs to make the layers infinitely wide.
```